Description: Example file to show the usage of the module
 python-xtermcolor (1.2-1) unstable; urgency=low
 .
   * Initial release. (Closes: #695210)
Author: Salvo 'LtWorf' Tomaselli <tiposchi@tiscali.it>

--- /dev/null
+++ python-xtermcolor-1.2/test.py
@@ -0,0 +1,3 @@
+from xtermcolor import colorize
+
+print (colorize('Red on white',0xff0000,bg=0xffffff))
